docs/install/installing-istio.md

```diff
@@ -139,6 +139,101 @@ spec:
 EOF
 ```
 
+After you install the cluster local gateway, your service and deployment for the local gateway are both named `cluster-local-gateway`.
+You do not need to update the `config-istio` configmap under the `knative-serving` namespace, because Knative Serving can use
+the local gateway cluster-local-gateway by default.
+
+### Updating the `config-istio` configmap to use a non-default local gateway
+
+If you create a custom service and deployment for local gateway with a name other than `cluster-local-gateway`, you
+need to update gateway configmap `config-istio` under the `knative-serving` namespace.
+
+1. Edit the 1config-istio1 configmap:
+
+```shell
+kubectl edit configmap config-istio -n knative-serving
+```
+
+2. Replace the `local-gateway.knative-serving.cluster-local-gateway` field with the custom service. If you name both
+the service and deployment `custom-local-gateway` under the namespace `istio-system`, it should be updated to:
+
+```
+custom-local-gateway.istio-system.svc.cluster.local
+```
+
+If both the custom service and deployment are labeled with `custom: custom-local-gateway`, not the default
```

Comment:
I might have missed it, but why would they have this label? Did we talk about it some place?

Reply:
I should surely add "As an example" at the beginning. This demonstrates how users should change the label of the cluster-local-gateway into something else.